src/apps/favorites-list/FavoritesList.entry.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC } from "react";
+import { withConfig } from "../../core/utils/config";
+import { pageSizeGlobal } from "../../core/utils/helpers/general";
+import { withText } from "../../core/utils/text";
+import { withUrls } from "../../core/utils/url";
+import FavoritesList from "./FavoritesList";
+
+interface FavoritesListConfigEntryProps {
+  fbsBaseUrlConfig: string;
+}
+interface FavoritesListUrlEntryProps {
+  materialUrl: string;
+  searchUrl: string;
+  fbsBaseUrl: string;
+  materialOverdueUrl: string;
+  feesPageUrl: string;
+  publizonBaseUrl: string;
+  dplCmsBaseUrl: string;
+  coverBaseUrl: string;
+  materialBaseUrl: string;
+  fbiBaseUrl: string;
+}
```

Comment:
I think we should make a global interface for the urls somewhere [here](src/core/utils/types). We reuse them a lot of places.

Reply:
I think we should note it, so someone can do it at some point, but i think that a global generalization should be done in one go, and therefore is out of the scope of this PR